src/com/xilinx/rapidwright/timing/DelayModel.java

```diff
@@ -79,7 +77,7 @@ public interface DelayModel {
      * @return Logic delay in ps. Return -1 if the connection does not exist.
      * @throws  IllegalArgumentException if the given bel is not recognized by the model.
      */
-    public short getLogicDelay(short belIdx, String frBelPin, String toBelPin, int encodedConfig);
+    public Short getLogicDelay(short belIdx, String frBelPin, String toBelPin, int encodedConfig);
```

Comment:
For the `short` -> `Short` changes, does the Javadoc need to be updated to reflect the possibility of returning `null`?  Previously, `-1` indicated that a connection does not exist, what does it mean if it returns `null`?